scripts/g.extension/g.extension.py

```diff
@@ -2027,9 +2027,9 @@ def resolve_source_code(url=None, name=None):
     GitLab:
 
     >>> resolve_source_code('gitlab.com/JoeUser/GrassModule') # doctest: +SKIP
-    ('remote_zip', 'https://gitlab.com/JoeUser/GrassModule/repository/archive.zip')
+    ('remote_zip', 'https://gitlab.com/JoeUser/GrassModule/'/-/archive/master/GrassModule-master.zip')
     >>> resolve_source_code('https://gitlab.com/JoeUser/GrassModule') # doctest: +SKIP
-    ('remote_zip', 'https://gitlab.com/JoeUser/GrassModule/repository/archive.zip')
+    ('remote_zip', 'https://gitlab.com/JoeUser/GrassModule/'/-/archive/master/GrassModule-master.zip')
```

Comment:
This seems to have a syntax error. Does the doctest run? I'm afraid it does not run as part of the testsuite, but it should be possible to execute using `g.extension --doctest` (a local hack).